website/project/metadata/schemas.py

```diff
@@ -24,5 +18,8 @@ def from_json(fname):
     ensure_schema_structure(from_json('osf-standard-1.json')),
```

Comment:
Small non-blocking aside, but in line 13, we should use a context manager to open and read the file. (to reduce the chance of data loss if the program crashes unexpectedly... yeah, this has actually happened to me)

```python
with open(os.path.join(here, fname)) as f:
    content = json.load(f)
return content
```